test/integration/admin_edits_item_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class AdminEditsItemTest < ActionDispatch::IntegrationTest
+
+  test 'admin can edit an item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.